sdk/maps/azure-maps-search/src/main/java/com/azure/maps/search/models/BaseSearchOptions.java

```diff
@@ -29,135 +34,228 @@ public BaseSearchOptions() {
     }
 
     /**
-     * Returns the top value.
-     * @return the top value.
+     *
```

Comment:
For all manually written Javadocs please include this section known as the `Description`. In newer versions of Java this is a requirement for Javadocs to be valid.


```suggestion
     * Sets the maximum number of responses that will be returned.
```

When viewing Javadocs this will also be the most prominent section, so having these will result in a better developer experience as well. It is also a good section to go into details on specific minutia about the given API or how the API affects wider portions of an application.